beta/project-griffon/using-project-griffon/versions-view.md

```diff
@@ -0,0 +1,27 @@
+# Versions Assurance View
+
+## Overview
+
+You may use the Extensions Version view in order to quickly and easily view which AEP extensions you have installed and if they are up to date in a client connected to a Project Griffon session. 
```

Comment:
`connect to an Assurance (also known as Project Griffon) session.` 
Then for all subsequent occurrences let's use Assurance.